spec/lib/openstax/exchange/client/shared_examples_for_exchange_client_v1.rb

```diff
@@ -0,0 +1,209 @@
+require 'spec_helper'
+
+RSpec.shared_examples "exchange client api v1" do
```

Comment:
I'm totally going to use this syntax in Exchange now
Much better than what I've been doing in spec/support